### PR TITLE
Fix snowflake ID in dbt extractor by adding [ch3428]

### PR DESCRIPTION
**Pull request by alyiwang**
### What?

- Add account into dataset id in DBT extractor 

### Checklist

- [ ] I have tested that the changes in this PR work as expected
- [x] I have added/updated tests that exercise the critical code paths in this diff

### CODE REVIEW

**Comment by mars-lan** [comment]
Fix title?

File: metaphor/dbt/extractor.py [hunk 1]
```diff
@@ -39,6 +39,9 @@ class DbtRunConfig(RunConfig):
     manifest: str
     catalog: str
 
+    # the database service account this DBT project is connected to
+    account: str = None
```
Comment (mars-lan) [inline metaphor/dbt/extractor.py:43]: This is easier but theoretically you can get it directly from `profile.yml`: https://github.com/MetaphorData/dbt/blob/main/profiles.yml#L5

**Comment by mars-lan** [comment]
> Fix title?

Never mind. I misread it.

File: metaphor/dbt/extractor.py (hunk 1 above)
Comment (alyiwang) [reply metaphor/dbt/extractor.py:43]: Right, the `profile` has all those information. But that way we need to bring `profile.yml` to the crawlers, besides the current manifest and catalog. Also, from the customer's perspective, they might prefer not to pass the `profile` as it contains passwords which the crawler doesn't need.

Comment (mars-lan) [reply metaphor/dbt/extractor.py]: Makes sense.

**Review by mars-lan — APPROVED** [review]

**Pull request merged**